scripts/deploy-helm.sh

```diff
@@ -71,7 +71,7 @@ check_dynatrace_docker_login() {
 
 print_help() {
   printf "
-usage: deploy-helm.sh [--role-name ROLE_NAME] [--create-autopilot-cluster] [--autopilot-cluster-name CLUSTER_NAME] [--upgrade-extensions] [--auto-default] [--quiet]
+usage: deploy-helm.sh [--role-name ROLE_NAME] [--create-autopilot-cluster] [--autopilot-cluster-name CLUSTER_NAME] [--without-extension-upgrade] [--auto-default] [--quiet]
```

Comment:
without-extensions-upgrade :D

Reply:
Ah, right, copied it like that everywhere... changing it :)